Remotion/ObjectBinding/Web/UI/Controls/BocDateTimeValueImplementation/Rendering/BocDateTimeValueRenderer.cs

```diff
@@ -213,7 +213,7 @@ private void RenderEditModeControls (BocDateTimeValueRenderingContext renderingC
       if (hasDateField)
       {
         renderingContext.Writer.AddAttribute (
-            HtmlTextWriterAttribute.Class, CssClassDateInputWrapper + " " + GetPositioningCssClass (renderingContext, DateTimeValuePart.Date));
+            HtmlTextWriterAttribute.Class, CssClassDateInputWrapper + " " + CssClassDefinition.RemotionThemed + " " + GetPositioningCssClass (renderingContext, DateTimeValuePart.Date));
```

Comment:
If we define the property on the base renderer (see BocRendererBase), we can use the property here.